src/openga.hpp

```diff
@@ -1075,7 +1076,17 @@ class Genetic
 		for(int i=0;i<N;i++)
 			gen.sorted_indices.push_back(i);
 
-		quicksort_indices_SO(gen.sorted_indices,gen,0,int(gen.sorted_indices.size())-1);
+		if (use_quick_search) {
+			quicksort_indices_SO(gen.sorted_indices,gen,0,int(gen.sorted_indices.size())-1);
+		} else {
```

Comment:
The standard style for this library is the Allman coding style.

Reply:
fixed